modules/auth/proc-authenticationg-with-the-guest-user.adoc

```diff
@@ -0,0 +1,15 @@
+[id="authenticating-with-the-guest-user"]
```

Comment:
[id="authenticating-with-the-guest-user_{context}"]

See https://redhat-documentation.github.io/modular-docs/#module_anchor-and-file-names-concept > Anchors